### ISSUE TITLE
Nuxt not starting because of error during module initialization

### ISSUE THREAD

**Issue by sbeugen** [artmizu/nuxt-prometheus#21]
Nuxt 3.6.5 
@artmizu/nuxt-prometheus: 2.1.3

I get the following error during `npm install` or `build` or `dev`, etc.
```
Error while requiring module @artmizu/nuxt-prometheus: Error: Cannot find module '/home/runner/work/nuxt-prometheus/nuxt-prometheus/node_modules/.pnpm/jiti@1.19.1/node_modules/jiti/lib/index.js'
```

**Comment by Flinty916** [comment]
Also experiencing this

**Comment by artmizu** [comment]
@sbeugen @Flinty916 

hello
my apologies for the inconvenience. i have fixed the issues in version 2.1.5 of the package. could you please check it?

**Comment by sbeugen** [comment]
Hi. No problem. Just had to pin an older version.

With 2.1.5 it works again. Thx.
